### DISCUSSION TITLE
scroll down limit.

### DISCUSSION THREAD

**Discussion by TinyProbe** [helix-editor/helix#6403] [Ideas] [answered]
I am a person who has been using vim and decided to use this. When scrolling down in Vim, it is possible to scroll down past the last line. Maybe this feature is here? If not, I'd like to be able to choose whether to scroll beyond the last line at the config file.

**Comment by the-mikedavis** [comment, marked as answer]
There is https://github.com/helix-editor/helix/pull/4590 which changes the scroll behavior at the end of the file slightly but I'm not sure that's what you're talking about.

We don't support putting the cursor past the last column or line in a document. Cursors in Helix are always selections so they must point to parts of the document that exist.

**Reply by TinyProbe** [reply]
When I received the source code of Helix Editor and built it, the features I mentioned were added. I guess I can use this.

For example, if the screen is (H : 10, W : 10) and the content of the document is 10 lines, the first to tenth lines of the document are displayed. Here, I wanted to scroll down to display the second to tenth lines of the document and have a blank space appear below them, but scrolling down below the last line did not work.

That's why I was hoping there would be similar to vim's screen scrolling.

**Comment by TinyProbe** [comment]
I think it's ok. thank you.

**Comment by gabydd** [comment]
You can scroll just the screen with ``zj`` and ``zk`` as well as some other binds under ``z``

See view mode: https://docs.helix-editor.com/master/keymap.html#view-mode

**Reply by TinyProbe** [reply]
Thanks for your reply.

In my case I want to go line by line.
